### PR TITLE
Change sign of pitch angle in debug menu

### PR DESCRIPTION
**Pull request by ClobberXD**
Tested; works

### CODE REVIEW

File: src/client/gameui.cpp [hunk 1]
```diff
@@ -128,7 +128,7 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< ", " << (player_position.Z / BS)
 			<< "), yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
 			<< yawToDirectionString(cam.camera_yaw)
-			<< ", pitch: " << (wrapDegrees_180(cam.camera_pitch)) << "°"
+			<< ", pitch: " << -(wrapDegrees_180(cam.camera_pitch)) << "°"
```
Comment (DS-Minetest) [inline src/client/gameui.cpp]: Wouldn't it be better to do the following?
```suggestion
			<< ", pitch: " << (-wrapDegrees_180(cam.camera_pitch)) << "°"
```

**Comment by paramat** [comment]
Does this actually match the result of 'get look vertical'? Where upward pitches seem to be negative:
```
* `get_look_vertical()`: pitch in radians
    * Angle ranges between -pi/2 and pi/2, which are straight up and down
      respectively.
```

**Comment by paramat** [comment]
Yeah i support doing this. The API values use radians so it's clear those values are a different thing. Negative for upwards pitch is so weird i think that has priority.

File: src/client/gameui.cpp (hunk 1 above)
Comment (paramat) [reply src/client/gameui.cpp]: I agree, i'm not sure if the outer brackets are needed, but it's clear they are meant to contain the value, so the negative sign is best inside.
+1 if done.

Comment (ClobberXD) [reply src/client/gameui.cpp]: Done

**Comment by ClobberXD** [comment]
The reversed signs when using the API methods is correct, IMHO, as MT uses the ~~upside-down~~ right-handed coordinate system. But the same isn't the best approach for pitch in the debug menu, as we're used to refer to looking up and looking down as increasing pitch and decreasing pitch respectively.

**Comment by paramat** [comment]
Thanks :+1:

**Comment by paramat** [comment]
Looks like @SmallJoker supports this being done so will merge.

**Pull request merged**